projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -120,20 +132,20 @@ export const withAIAssistant = createHigherOrderComponent(
 			);
 
 			removeBlocks( otherBlocksIds );
-		}, [ blocks, blockType, content, replaceBlock, clientIds, removeBlocks ] );
+		}, [ blockType, replaceBlock, removeBlocks ] );
 
 		const blockControlProps = {
 			group: 'block',
 		};
-		const rawContent = getRawTextFromHTML( content );
+		// const rawContent = getRawTextFromHTML( content );
 
 		return (
 			<>
 				<BlockEdit { ...props } />
 
 				<BlockControls { ...blockControlProps }>
 					<AiAssistantDropdown
-						disabled={ ! rawContent?.length }
+						disabled={ false }
 						onChange={ handleChange }
 						onReplace={ replaceWithAiAssistantBlock }
```

Comment:
Why can `replaceWithAiAssistantBlock` and `handleChange` not be moved into `AiAssistantDropdown`?